software/wikidocs.yml

```diff
@@ -0,0 +1,14 @@
+name: "WikiDocs"
+website_url: "http://wikidocs.it"
+source_code_url: "https://github.com/Zavy86/WikiDocs"
+description: "A databaseless markdown flat-file wiki engine."
+licenses:
+  - MIT
+platforms:
+  - PHP
+  - Docker
+# list of tags (categories), see https://github.com/awesome-selfhosted/awesome-selfhosted-data/tree/master/tags for the full list of tags
+tags:
+  - Wikis
+depends_3rdparty: false
```

Comment:
This line can be removed as its optional

Reply:
Oops. I think I went through and removed all of the comments first and then missed that line as being optional when I did further cleanup. Bad workflow approach on my part. Fixed.